docs/Using-Fleet/fleetctl-CLI.md

```diff
@@ -189,6 +190,95 @@ spec:
   query: select * from processes
 ```
 
+## Using fleetctl with an API-only user
+
+Now that `fleetctl` and the Fleet server are configured, it can be helpful to create an API-only user to use when running automated workflows. An API-only user can be given a role based on the abilities it needs. The default access level is `Observer`. For more information on roles, see the [user permissions documentation](./Permissions.md#user-permissions).
```

Comment:
Love that you point the user to permissions docs here.

Reply:
Thanks! My goal is to always try to give as much info as I can in place without duplicating too much info across the docs.